utils.py

```diff
@@ -53,6 +57,30 @@ def yaml_helper(fpath):
     return result
 
 
+def xml_helper(xml_path):
+    with open(xml_path, 'r') as f:
+        xml_str = f.read()
+    json = xmltodict.parse(xml_str)
+    return json
+
+
+def json_helper(json_path):
+    with open(json_path, 'r', encoding='UTF-8') as f:
+        return json.loads(f.read())
+
+
+@data.route('/json_test')
+def json_test():
+    path = os.path.join("/root/api-server/", "test.json")
+    return json_helper(path)
+
+
+@data.route('/xml_test')
```

Comment:
Thanks for adding these test cases. Ideally and personally I'd like to see them be put in `test.py` and being tested by CI/CD with `pytest`, instead of working as real endpoints. But I'm OK with them living here for now since we aren't using them anyway.

Reply:
Thanks, I agree with you